docs/references/faq/faq.md

```diff
@@ -87,6 +87,18 @@ Network volumes are billed on a per-hour basis. Please note that if your machine
 RunPod is also not designed to be a cloud storage system; storage is provided in the pursuit of running tasks using its GPUs, and not meant to be a long-term backup solution.
 It is highly advisable to continually back up anything you want to save offsite locally or to a cloud provider.
 
+
+### Can I withdraw my unused balance?
+
+No, RunPod does not offer the option to withdraw your unused balance after depositing funds into your account. When you add funds to your RunPod account, these credits are non-refundable and can only be used for RunPod services.
+
+:::important
```

Comment:
Some code formatters like Prettier want to see blank lines around these. See https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier

Not a problem for this PR. Just something to think about when formatting your markdown.